packages/core/src/lib/employee-availability/employee-availability.service.ts

```
@@ -0,0 +1,15 @@
+import { MikroOrmEmployeeAvailabilityRepository } from './repository/micro-orm-employee-availability.repository';
+import { TenantAwareCrudService } from './../core/crud';
+import { EmployeeAvailability } from './employee-availability.entity';
+
+@Injectable()
+export class EmployeeAvailabilityService extends TenantAwareCrudService<EmployeeAvailability> {
+	constructor(
+		typeOrmEmployeeAvailabilityRepository: TypeOrmEmployeeAvailabilityRepository,
+		mikroOrmEmployeeAvailabilityRepository: MikroOrmEmployeeAvailabilityRepository
+	) {
+		super(typeOrmEmployeeAvailabilityRepository, mikroOrmEmployeeAvailabilityRepository);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)